api/v1beta1/nodemaintenance_webhook.go

```diff
@@ -96,7 +96,7 @@ func (r *NodeMaintenance) SetupWebhookWithManager(mgr ctrl.Manager) error {
 }
 
 // TODO(user): change verbs to "verbs=create;update;delete" if you want to enable deletion validation.
-//+kubebuilder:webhook:path=/validate-nodemaintenance-medik8s-io-v1beta1-nodemaintenance,mutating=false,failurePolicy=fail,sideEffects=None,groups=nodemaintenance.medik8s.io,resources=nodemaintenances,verbs=create;update,versions=v1beta1,name=vnodemaintenance.kb.io,admissionReviewVersions={v1,v1beta1}
+//+kubebuilder:webhook:path=/validate-nodemaintenance-medik8s-io-v1beta1-nodemaintenance,mutating=false,failurePolicy=fail,sideEffects=None,groups=nodemaintenance.medik8s.io,resources=nodemaintenances,verbs=create;update,versions=v1beta1,name=vnodemaintenance.kb.io,admissionReviewVersions=v1
```

Comment:
I have some doubts regarding this change, after reading `if you are not using the v1beta1 K8s APIs which are no longer supported from k8s 1.22 version` in [here](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.14.0/#upgrade-k8s-versions-to-use-122-golangv3).
node-maintenance-operator API is [`v1beta1`](https://github.com/medik8s/node-maintenance-operator/blob/main/api/v1beta1/nodemaintenance_types.go#L17), is that a problem or this change is valid since it is not k8s API?

Reply:
the latter, this is about the admissionreview version, not NMO